src/pages/library/DataBrowserDetails.js

```diff
@@ -194,17 +195,21 @@ export const SidebarComponent = ({ dataObj, id }) => {
         ])
       ]),
       h(ButtonPrimary, {
-        disabled: true,
-        tooltip: 'We are currently working on the link to workspace feature which will be available soon.',
+        disabled: dataObj.access !== datasetAccessTypes.GRANTED,
+        tooltip: dataObj.access === datasetAccessTypes.GRANTED ? '' : uiMessaging.controlledFeature_tooltip,
```

Comment:
I know it wasn't introduced by your PR but can we rename `controlledFeature_tooltip` as `controlledFeatureTooltip` to be consistent with the `camelCase` convention we follow?